.github/workflows/pull-requests.yml

```diff
@@ -17,11 +17,11 @@ jobs:
         uses: denolib/setup-deno@master
         with:
           deno-version: 1.34.1
-      - name: Cache Deno dependencies
-        uses: actions/cache@v2
-        with:
-          path: ${{ env.DENO_DIR }}
-          key: ${{ hashFiles('deno.lock') }}
+      # - name: Cache Deno dependencies
+      #   uses: actions/cache@v2
+      #   with:
+      #     path: ${{ env.DENO_DIR }}
+      #     key: ${{ hashFiles('deno.lock') }}
```

Comment:
Was this causing issues? I'm actually not 100% sure if it was working because it looked like deps were still being downloaded on each of my pushes so I don't really mind commenting this out for now 😛

Reply:
I commented this a little while ago to try to see if it was impacting a different issue and must have forgotten a TODO. I uncommented this and it looks fine. Thanks for catching that